doc/api-docs/conf.py

```diff
@@ -12,33 +12,26 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
+from datetime import datetime
 import importlib
 import itertools
 import os
 import pkgutil
 import subprocess
 import sys
-
-from datetime import datetime
-from sphinx.ext import apidoc
 from unittest.mock import MagicMock
 
+import natcap.invest
+from sphinx.ext import apidoc
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 DOCS_SOURCE_DIR = os.path.dirname(__file__)
 INVEST_ROOT_DIR = os.path.join(DOCS_SOURCE_DIR, '..', '..')
 INVEST_BUILD_DIR = os.path.join(INVEST_ROOT_DIR, 'build')
```

Comment:
Are these used anywhere else now that we're properly defining `INVEST_LIB_DIR`?  Might they be able to be removed?

Reply:
You're right, they can be removed